creature.cpp

```diff
@@ -138,6 +139,7 @@ CreatureAction Creature::castSpell(Spell* spell) const {
     return CreatureAction("You can't cast this spell yet.");
   return CreatureAction(this, [=] (WCreature c) {
     c->addSound(spell->getSound());
+    fx::spawnEffect("circular_blast", c->getPosition().getCoord());
```

Comment:
I prefer to couple rendering and gameplay through the View interface, but I won't bother you with this, I'll take care of it later.